test/test.js

```diff
@@ -52,284 +58,236 @@ describe('YeldaChat', () => {
       assert.typeOf(yeldaChat.init, 'function', 'init function exists')
     })
 
-    it('should return null if assistantId or assistantSlug are not passed', () => {
+    it('should webChatContainer is null if assistantId or assistantSlug are not passed', async () => {
       const mockData = {
         'locale': 'fr_FR'
       }
 
-      const response = yeldaChat.init(mockData)
-      assert.typeOf(response, 'null', 'init function returns null')
-    })
-
-    it('should return nothing(undefined) if required data passed', () => {
-      const mockData = {
-        'assistantSlug': 'sodebo',
-        'assistantId': '12345678'
-      }
-
-      const response = yeldaChat.init(mockData)
-      assert.typeOf(response, 'undefined', 'init function returns undefined')
+      yeldaChat.unLoadChat()
+      await yeldaChat.init(mockData)
+      expect(yeldaChat.webChatContainer).to.be.null
     })
 
-    it('should create webChatContainer DOM element if required data are passed', () => {
+    it('should create webChatContainer DOM element if required data are passed', async () => {
       const mockData = {
         'assistantSlug': 'sodebo',
         'assistantId': '12345678'
       }
-
-      yeldaChat.init(mockData)
+      yeldaChat.unLoadChat()
+      await yeldaChat.init(mockData)
       expect(yeldaChat.webChatContainer).not.to.be.null
     })
   })
 
   describe('yeldaChat.setupChat', () => {
-
     describe('yeldaChat.setupChat initial checks', () => {
-      it('should return null if assistantId is missing', () => {
+      it('should webChatContainer is null if assistantId is missing', async () => {
```

Comment:
```suggestion
      it('should set webChatContainer to null if assistantId is missing', async () => {
```

i let you edit it in other places ("should webChatContainer is null" is not correct English)